packages/preview-middleware-client/src/adp/init.ts

```diff
@@ -27,6 +29,15 @@ export default function (rta: RuntimeAuthoring) {
             }
         }
     });
+    
+    const ui5Version = (await VersionInfo.load()) as { version: string };
+    const ui5VersionValidationMessage = getUI5VersionValidationMessage(ui5Version.version);
+
+    if(ui5VersionValidationMessage) {
+        sendAction(showMessage('The current SAPUI5 version set for this Adaptation project is 1.70. The minimum version to use for SAPUI5 Adaptation Project and its SAPUI5 Visual Editor is 1.71'));
```

Comment:
Please replace hardcoded test with the `ui5VersionValidationMessage` variable 😉

Reply:
Fixed